.github/workflows/bi-connectors.yml

```diff
@@ -0,0 +1,60 @@
+name: Build connectors for BI tools
+
+on:
+  push:
+    paths:
+      - 'bi-connectors/PowerBIConnector/**'
+      - 'bi-connectors/TableauConnector/**'
+      - '.github/workflows/bi-connectors.yml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Pack Tableau JDBC connector
+      id: pack-tableau-jdbc
+      run: |
+        zip -r opensearch_sql_jdbc.taco . -x *.taco
+      working-directory: bi-connectors/TableauConnector/opensearch_sql_jdbc
+    - name: Prepare Power BI ODBC connector
+      run: |
+        cp OpenSearch.pq OpenSearch.m
+      working-directory: bi-connectors/PowerBIConnector
+    - name: Pack Power BI ODBC connector
+      id: pack-powerbi-odbc-os-proj
+      run: |
+        zip OpenSearchProject.mez *.png *.m *.resx *.pqm
+      working-directory: bi-connectors/PowerBIConnector
+    - name: Prepare Power BI ODBC connector for 'Amazon OpenSearch Service'
+      id: prep-powerbi-odbc-amz-os-svc
+      run: |
+        sed -i 's/<value>OpenSearch Project<\/value>/<value>Amazon OpenSearch Service<\/value>/g' resources.resx
+        sed -i 's/Documentation.Name = "OpenSearch Project"/Documentation.Name = "Amazon OpenSearch Service"/g' OpenSearch.m
+        sed -i 's/section OpenSearch;/section AmazonOpenSearchConnector;/' OpenSearch.m
+        sed -i 's/\[DataSource.Kind="OpenSearch", Publish="OpenSearch.Publish"\]/[DataSource.Kind="AmazonOpenSearchConnector", Publish="AmazonOpenSearchConnector.Publish"]/g' OpenSearch.m
+        sed -i 's/OpenSearch.Contents/AmazonOpenSearchConnector.Contents/g' OpenSearch.m
+        sed -i 's/OpenSearch.Publish/AmazonOpenSearchConnector.Publish/g' OpenSearch.m
+        sed -i 's/OpenSearch.Icons/AmazonOpenSearchConnector.Icons/g' OpenSearch.m
+        sed -i 's/OpenSearchImpl/AmazonOpenSearchConnectorImpl/g' OpenSearch.m
+        sed -i 's/OpenSearchType/AmazonOpenSearchConnectorType/g' OpenSearch.m
+        sed -i 's/OpenSearch = \[/AmazonOpenSearchConnector = [/' OpenSearch.m
+        sed -i 's/"opensearch-sql"/"Amazon-OpenSearch-Service"/g' OpenSearch.m
```

Comment:
Do we need this to change? Seems like it's not the right replacement.

Reply:
Fixed in b287841c.